asf_tools/dem.py

```diff
@@ -78,7 +78,7 @@ def shift_for_antimeridian(dem_file_paths: List[str], directory: Path) -> List[s
 def prepare_dem_vrt(vrt: Union[str, Path], geometry: Union[ogr.Geometry, shapely.geometry.GeometryCollection]):
     """Create a DEM mosaic VRT covering a given geometry
 
-    The DEM mosaic is assembled from the Copernicus GLO-30 Public DEM tiles that intersect the geometry.
```

Comment:
I dropped the "Public" because Copernicus has dropped it from their product name, but I am waffling on weather it's worth having on its own.

Reply:
Also, need to check everywhere for this